utils/fasm/src/fasm.cpp

```diff
@@ -58,8 +58,9 @@ void FasmWriterVisitor::visit_clb_impl(ClusterBlockId blk_id, const t_pb* clb) {
       std::vector<std::string> fasm_prefixes = vtr::split(prefix_unsplit, " ");
       if(fasm_prefixes.size() != static_cast<size_t>(blk_type_->capacity)) {
         vpr_throw(VPR_ERROR_OTHER,
-                  __FILE__, __LINE__, "fasm_prefix = %s, capacity = %d",
-                  prefix_unsplit.c_str(), blk_type_->capacity);
+                  __FILE__, __LINE__,
+                  "number of fasm_prefix options(%d) must match capacity(%d)",
+                  prefix_unsplit.size(), blk_type_->capacity);
```

Comment:
You took the wrong size?  You are printing the size of the string, not the number of prefixes?